Makefile

```diff
@@ -106,3 +106,21 @@ proto: install-tools ## Generate protobuf files
 .PHONY: bench-kafka
 bench-kafka:
 	go test -count 1 -run BenchmarkCompression -bench=. -benchmem ./services/streammanager/kafka/client
+
+.PHONY: check-swagger
+check-swagger:
+	which swagger || (GO111MODULE=off go get -u github.com/go-swagger/go-swagger/cmd/swagger)
+
+.PHONY: swagger
+swagger: check-swagger
+	GO111MODULE=on go mod vendor  && GO111MODULE=off swagger generate spec -o ./swagger.yaml --scan-models
+
+.PHONY: serve-swagger
+serve-swagger: check-swagger
+	go install github.com/go-swagger/go-swagger/cmd/swagger
+	swagger serve ./swagger.yaml
+
```

Comment:
We don't need these right?

Reply:
Don't we need it for installations ?

Reply:
This would be sufficient. `go install github.com/go-swagger/go-swagger/cmd/swagger` which we will be added in `install-tools`.

Reply:
done.